### DISCUSSION TITLE
Is there any way to get quickplot to only plot part of the solution?

### DISCUSSION THREAD

**Discussion by cyasing** [pybamm-team/PyBaMM#2938] [Q&A]
I am using an experiment, whose initial 1.5 hours of time is not necessary to be plotted, just the remaining time. Can this be done?
Something like this: `solutionch = solution[125:]` where 125 is for the initial 125 entries of time.

**Comment by brosaplanella** [comment]
Yes, you can access the entries of a solution variable, e.g. `solution["Voltage [V]"].entries[125:]`

**Reply by cyasing** [reply]
Right. I knew about this method, I was just wondering how to get this in the format of quickplot. Couldn't figure out the syntax.

**Reply by brosaplanella** [reply]
Not sure if using the `variable_limits` argument (https://pybamm.readthedocs.io/en/latest/source/api/plotting/quick_plot.html) would work. You can give at a go and see if it works.
